admin/pages/advanced.php

```diff
@@ -48,8 +48,13 @@
 
 echo '<h2 class="nav-tab-wrapper">';
 foreach ( $tabs->get_tabs() as $tab ) {
-	$active = ( $tabs->is_active_tab( $tab ) ) ? ' nav-tab-active' : '';
-	echo '<a class="nav-tab' . $active . '" id="' . $tab->get_name() . '-tab" href="' . esc_url( admin_url( 'admin.php?page=wpseo_advanced&tab=' . $tab->get_name() ) ) . '">' . $tab->get_label() . '</a>';
+	printf(
+		'<a class="nav-tab %1$s" id="%2$s" href="%3$s">%4$s</a>',
+		( $tabs->is_active_tab( $tab ) ? 'nav-tab-active' : '' ),
+		esc_attr( $tab->get_name() . '-tab' ),
+		esc_url( admin_url( 'admin.php?page=wpseo_advanced&tab=' . $tab->get_name() ) ),
+		$tab->get_label()
```

Comment:
Expecting `$tab->get_label()` to be escaped with `esc_html` as well.

Reply:
See my response: https://github.com/Yoast/wordpress-seo/pull/8378/files#r155085616